UI/window-basic-main.cpp

```diff
@@ -1507,7 +1507,8 @@ bool OBSBasic::InitBasicConfigDefaults()
 				  "StreamAudioEncoder", "aac");
 	config_set_default_string(basicConfig, "SimpleOutput",
 				  "RecAudioEncoder", "aac");
-
+	config_set_default_uint(basicConfig, "SimpleOutput", "SimpleRecTracks",
+				(1 << 0));
```

Comment:
The blank space here visually separated Simple Output and Advanced Output settings. Re-add it below the new Simple Output setting to keep them separated.

Reply:
done